pkgs/development/python-modules/protocol/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub }:
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
```

Comment:
not needed

Reply:
as in `name` will be autogenerated from pname/pversion ? I copy/pasted that from python-modules.

Reply:
In case of `buildPython*`, yes

Reply:
I removed the line and force pushed